hydrator-common/src/main/java/co/cask/hydrator/common/MockPipelineConfigurer.java

```diff
@@ -26,17 +26,23 @@
 import co.cask.cdap.etl.api.PipelineConfigurer;
 import co.cask.cdap.etl.api.StageConfigurer;
 
+import java.util.HashMap;
+import java.util.Map;
 import javax.annotation.Nullable;
 
 /**
  * Mock test configurer used to test configure pipeline's validation of input schema and setting of output shcema.
  */
 public class MockPipelineConfigurer implements PipelineConfigurer {
-  private final Schema inputSchema;
+  private final Map<String, Schema> inputSchemas = new HashMap<String, Schema>();
   Schema outputSchema;
 
   public MockPipelineConfigurer(Schema inputSchema) {
-    this.inputSchema = inputSchema;
+    this.inputSchemas.put("", inputSchema);
```

Comment:
I think so since none of the tests are using getInputSchema()/getInputSchemas().